modules/test/playwright/pages/commerce/commerce-order-content-web/placedOrdersPage.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FrameLocator, Locator, Page} from '@playwright/test';
+
+import {CommerceLayoutsPage} from '../commerceLayoutsPage';
+
+export class PlacedOrdersPage {
+	readonly billingAddress: Locator;
+	readonly configurationIFrame: FrameLocator;
+	readonly configurationIFrameShowFullAddressToggle: Locator;
+	readonly configurationIFrameShowPhoneNumberToggle: Locator;
+	readonly configurationIFrameSaveButton: Locator;
+	readonly configurationMenuItem: Locator;
+	readonly layoutsPage: CommerceLayoutsPage;
+	readonly optionsButton: Locator;
+	readonly orderItemActionsButton: Locator;
+	readonly orderItemActionsButtonEdit: Locator;
+	readonly page: Page;
+	readonly pageLabel: Locator;
+	readonly pageTitle: Locator;
+	readonly panelList: Locator;
+	readonly shippingAddress: Locator;
+	readonly viewButton: Locator;
+
+	constructor(page: Page) {
+		this.billingAddress = page.getByTestId('commerceBillingAddress');
+		this.configurationIFrame = page.frameLocator(
+			'iframe[id="modalIframe"]'
+		);
+		this.configurationIFrameShowFullAddressToggle =
+			this.configurationIFrame.getByLabel('Show Full Addresses');
+		this.configurationIFrameShowPhoneNumberToggle =
+			this.configurationIFrame.getByLabel('Show Phone Number');
+		this.configurationIFrameSaveButton = this.configurationIFrame.getByRole(
+			'button',
+			{name: 'Save'}
+		);
+		this.configurationMenuItem = page.getByRole('menuitem', {
+			exact: true,
+			name: 'Configuration',
+		});
+		this.layoutsPage = new CommerceLayoutsPage(page);
+		this.optionsButton = page
+			.locator(
+				'#portlet-topper-toolbar_com_liferay_commerce_order_content_web_internal_portlet_CommerceOrderContentPortlet'
```

Comment:
can you try to replace this to 
`portlet_com_liferay_commerce_order_content_web_internal_portlet_CommerceOrderContentPortlet` and check if it works? I prefer this id instead of the used one